content-src/asrouter/components/ModalOverlay/_ModalOverlay.scss

```diff
@@ -20,7 +21,7 @@
 
 .modalOverlayInner {
   width: 960px;
-  height: 570px;
+  // height: 570px;
```

Comment:
Removing the height makes the older onboarding kinda broken (see `ONBOARDING_1 ` message in about:newtab#devtools). We might be removing that though... let me ask about that

Also, if you are removing the fixed height you probably also want some kind of `top` value that isn't `top: calc(50% - 285px);` (i.e. based on a height of 570)

Reply:
Yeah, so we are still keeping this around for now; do you think you could put it in a separate class? Steps to preview this UI: 
![image](https://user-images.githubusercontent.com/1455535/58277928-d4b1be80-7d68-11e9-88e1-d5bb8b4cdc5a.png)

Reply:
Durrr I keep forgetting to check on dark mode. :P I'll tweak things, and make the `top` positioning more dynamic.